frontend/app/configuration/Config.scala

```diff
@@ -159,11 +159,11 @@ object Config {
       codes = PromoCodeSet(PromoCode("EH2016")),
       description = "Free English Heritage membership worth £88 when you become a Partner or Patron Member",
       expires = DateTime.parse("2016-04-01T01:00:00Z"),
-      imageUrl = "http://lorempixel.com/400/200/abstract",
+      imageUrl = "https://s3-eu-west-1.amazonaws.com/memsub-promo-images/eh2016.png",
```

Comment:
Think this approach might require a bit more thought - perhaps a chat with another frontend developer